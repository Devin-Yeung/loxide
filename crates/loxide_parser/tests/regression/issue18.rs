regression!(issue18, |src| {
    let mut parser = ::loxide_parser::parser::Parser::new(src);
    let result = parser.parse();
    insta::assert_debug_snapshot!(result)
});
