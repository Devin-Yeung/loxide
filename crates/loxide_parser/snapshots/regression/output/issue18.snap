---
source: crates/loxide_parser/tests/regression/issue18.rs
expression: result
input_file: crates/loxide_parser/snapshots/regression/snapshots-inputs/issue18.lox
---
(
    [
        PrintStmt(
            Expr {
                kind: Variable(
                    Variable {
                        name: "a",
                    },
                ),
            },
        ),
    ],
    [],
)
